census_data_aggregator/__init__.py

```diff
@@ -79,6 +79,9 @@ def approximate_median(range_list, design_factor=None):
             variance of the dataset. The Census Bureau publishes design factors as part of its PUMS Accuracy statement.
             Find the value for the dataset you are estimating by referring to `the bureau's reference material`_.
             If you do not provide this input, a margin of error will not be returned.
+        sampling_percentage (float, optional): A statistical input used to correct variance for finite population.
```

Comment:
Can we use a more casual term than finite population? The percentage of the population that was in the survey sample.